streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -319,7 +321,7 @@ public void shouldRestoreToWindowedStores() throws IOException {
             changelogName,
             1,
             offset,
-            start,
+            end,
```

Comment:
"exposed" after rebase -- for time-windows, we want to use the end-timestamp (even if this change is not implemented yet, I added it here, to avoid touching this class later on again).